### ISSUE TITLE
Normalize image before inference

### ISSUE THREAD

**Issue by mihelich** [JunkyByte/easy_ViTPose#12]
Thanks for the helpful repo!

When using the pretrained models, should I normalize the image tensor before feeding it to the model?

For training, `COCODataset` applies the usual `Normalize` transform ([code](https://github.com/JunkyByte/easy_ViTPose/blob/main/easy_ViTPose/datasets/COCO.py#L124)) when loading images. The config files also include `NormalizeTensor` steps in `train_pipeline` and `val_pipeline`, though that part of the config doesn't appear to be used in this repo.

On the inference side, however, the `inference.py` script only scales the image input to `[0, 1]`, without normalizing ([code](https://github.com/JunkyByte/easy_ViTPose/blob/main/easy_ViTPose/datasets/COCO.py#L124)). Is that a bug, or am I missing something?

**Comment by JunkyByte** [comment]
Hello, thanks for your interest and for the bug report, I agree the preprocessing is different in inference, the original implementation I built on (https://github.com/jaehyunnn/ViTPose_pytorch) seems to contain this bug and I did not notice.

Should be fixed with c8a7a1460170e83a8120c1b9cf691776c6379b88, I inspected the inference results visually and they improve. I referenced this issue on the original implementation.
